### DISCUSSION TITLE
Exiting the editor via keyboard should always go through compose screen first

### DISCUSSION THREAD

**Discussion by LittleEndu** [ppy/osu#14746] [Ideas] [answered]
A lot of essential features that used to be windows in stable are now screens in lazer. The windows usually (but not always, very confusing) would close first when pressing <kbd>ESC</kbd> and then pressing <kbd>ESC</kbd> again would close the editor. In lazer it doesn't matter what screen you are on, editor would always exit.

**Comment by bdach** [comment]
why would this matter? just muscle memory? i'm not sure lazer should try and carbon copy stable editor UX to the dot. in fact it has very much deviated from it in places on purpose.

if the issue is that some changes may be lost due to this new behaviour, the _actual_ issue here is https://github.com/ppy/osu/issues/10314.

**Reply by LittleEndu** [reply]
Yeah, it's muscle memory for me. I'm so used to opening timing panel, doing the edits, then exiting via <kbd>ESC</KBD>. Same with song setup.

**Comment by peppy** [comment, marked as answer]
This is not an issue and not changing.
